project/Dependencies.scala

```diff
@@ -9,15 +9,15 @@ object Dependencies {
   // Runtime dependencies versions
   private val ExasolVersion = "6.1.7"
   private val ImportExportUDFVersion = "0.1.0"
```

Comment:
Could you please also update this? The latest should be should be `0.2.0`.
I think it already includes jackson-databind and jackson-module.